### ISSUE TITLE
C-137 vs Replacement Dimension Characters

### ISSUE THREAD

**Issue by Trendy** [Shmakov/Rick-and-Morty-Characters#3]
In your summary you list Jessica one time and separate out C-137 Jerry, Beth, and Summer as separate characters. You seem to have forgotten that the original Jessica character was from C-137 as well, and does not appear after Rick Potion #9 (https://en.wikipedia.org/wiki/Rick_Potion_No._9)

**Comment by Shmakov** [comment]
You are completely right. Same goes for Principal Vagina, Jessica's Friend, Brad and few others. That is the reason I list "Died in Dimension C-137" in their [death notes](https://github.com/Shmakov/Rick-and-Morty-Characters/blob/master/data.js#L365).

On the same note, isn't past/memories of those characters from C-137 is exactly the same as Replacement Dimension?

And in reality, I just didn't want to make the timeline too long :) I'm also open to the pull requests :)

**Comment by trammel-may** [comment]
We actually don't know if the original dimension we see is C-137. All we know is that Rick refers to himself as Rick C-137. We can infer that the original dimension we see is not Rick's original dimension given his memories of Morty as a baby where as Rick in the starting dimension was gone for 20 years (longer than Morty has been alive). Given that Rick also keeps his C-137 title after jumping dimensions (as seen in post Cronenberg episodes, and in the latest season it was revealed the jump dimensions again because of the squirrels) we can assume that C-137 is Rick's original dimension which excludes the episode 1 dimension from being C-137 (or at the very least excludes it from definitely being referred to as such). Many have opted to calling the first dimension the prime dimension and that dimension (our current) Morty as Morty prime as we cannot know if he is C-137. Most evidence points to evil or eyepatch Morty to being the original Morty (C-137)

**Comment by Shmakov** [comment]
Anyone knows GitHub usernames of Justin Roiland and Dan Harmon? ;) We need some answers here!

**Comment by trammel-may** [comment]
probably something to do with squanch

**Comment by dwarburt** [comment]
@tramstheman Although Morty does identify himself as "Morty C-137" in [Rikshank Redemption](http://rickandmorty.wikia.com/wiki/The_Rickshank_Rickdemption/Transcript).

> Hold your fire! Hold your fire! I'm Morty C-137.

**Comment by trammel-may** [comment]
@dwarburt Yes, he does, however I think it is because Rick refers to himself as Rick C-137. We never hear Rick refer to him as Morty C-137. I

**Comment by dwarburt** [comment]
@tramstheman if Rick called Morty "Morty C-137" it wouldn't prove much either. There is no good reason to think C-137 isn't the original dimension. Rick's memory of a baby Morty can be explained by so many other less exotic hypotheses like the very simple and obvious explanation that he visited the baby without Beth knowing about it. After all he is shown to try to hide his sentimentality, so portalling into the babies room for a quick peek and pickup and bounce and then out before anyone saw him is a pretty in character thing for him to do.
